### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by asponomar**

### CODE REVIEW

File: src/main/java/hw3/ex1/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        loginButton.click();
+    }
+
+    public String getUserNameText() {
+        return userName.getText();
+    }
+
+    public List<WebElement> getHeaderSectionMenuButtons() {
+        return headerSectionMenuButtons;
+    }
```
Comment (khda91) [inline src/main/java/hw3/ex1/HomePage.java]: This method could be moved to the `AbstractBasePage` class

File: src/main/java/hw3/ex1/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        return epamJdiGitHubLink;
+    }
+
+    public WebElement getLeftSection() {
+        return leftSection;
+    }
+
+    public WebElement getFooter() {
+        return footer;
+    }
```
Comment (khda91) [inline src/main/java/hw3/ex1/HomePage.java]: these methods could be moved to the `AstractBasePage` class

File: src/test/java/hw2/ex1/EpamJdiHomePageTest.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
+package hw2.ex1;
```
Comment (khda91) [inline src/test/java/hw2/ex1/EpamJdiHomePageTest.java]: Why is HW2 related file in the Pull Request for the HW3?

File: src/test/java/hw2/ex2/EpamJdiHomePageSoftAssertTest.java [hunk 4]
```diff
@@ -0,0 +1,161 @@
+package hw2.ex2;
```
Comment (khda91) [inline src/test/java/hw2/ex2/EpamJdiHomePageSoftAssertTest.java]: Why is HW2 related file in the Pull Request for the HW3?

File: src/test/java/hw3/AbstractBaseTest.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    @BeforeMethod
+    public void methodSetUp() {
+        driver = new ChromeDriver();
+
+        driver.manage().window().maximize();
+
+//        driver.manage().timeouts().implicitlyWait(5000, TimeUnit.MILLISECONDS);
+//        driver.manage().timeouts().pageLoadTimeout(7000, TimeUnit.MILLISECONDS);
+//        driver.manage().timeouts().setScriptTimeout(9000, TimeUnit.MILLISECONDS);
```
Comment (khda91) [inline src/test/java/hw3/AbstractBaseTest.java]: Why are these lines commented?

File: src/main/java/hw3/ex1/HomePage.java (hunk 1 above)
Comment (asponomar) [reply src/main/java/hw3/ex1/HomePage.java]: Moved

File: src/main/java/hw3/ex1/HomePage.java (hunk 2 above)
Comment (asponomar) [reply src/main/java/hw3/ex1/HomePage.java]: Moved

File: src/test/java/hw3/AbstractBaseTest.java (hunk 5 above)
Comment (asponomar) [reply src/test/java/hw3/AbstractBaseTest.java]: Forgot to uncomment before commit, commented to check test passing faster. Uncommented

File: src/test/java/hw2/ex1/EpamJdiHomePageTest.java (hunk 3 above)
Comment (asponomar) [reply src/test/java/hw2/ex1/EpamJdiHomePageTest.java]: Created branch based on hw2

File: src/test/java/hw2/ex2/EpamJdiHomePageSoftAssertTest.java (hunk 4 above)
Comment (asponomar) [reply src/test/java/hw2/ex2/EpamJdiHomePageSoftAssertTest.java]: Created branch based on hw2

**Pull request merged**